src/curl.c

```diff
@@ -448,6 +448,7 @@ static int swupd_curl_get_file_full(const char *url, char *filename,
 			break;
 		case CURLE_WRITE_ERROR:
 			fprintf(stderr, "Curl: Error downloading to local file - %s\n", filename);
+			fprintf(stderr, "Check free space for %s ?", state_dir);
```

Comment:
isn't this line missing the \n?

Reply:
also, is it ok to have a space between the path and the "?" ?

Reply:
Agreed on the \n, and there is one example already merged that doesn't use a space. I'll update to that standard.